### ISSUE TITLE
Question about ac

### ISSUE THREAD

**Issue by LuFan31** [ningkp/LfOSA#1]
Thank you for your excellent work! 
For the  'ac'  here, is it the logits output from the last fc layer of the network, or is it the value activated by ReLu or other activation function?
![图片](https://user-images.githubusercontent.com/88334335/179555451-e465b94c-fbba-4511-85e6-6ef6a82b436a.png)
![图片](https://user-images.githubusercontent.com/88334335/179555766-16ba1eb9-1bff-4235-8687-31598229913a.png)

**Comment by ningkp** [comment]
The 'ac' is the logits output from the last fc layer of the network.
